google-cloud-vision/lib/google/cloud/vision/v1/image_annotator_client.rb

```diff
@@ -136,7 +136,7 @@ def initialize \
               updater_proc = credentials.updater_proc
             end
 
-            package_version = Gem.loaded_specs['google-cloud-vision'].version.version
+            package_version = Gem.loaded_specs['google-cloud'].version.version
```

Comment:
Seems to be an naming issue in generation. It also generates a google-cloud.gemspec.